test/correctness/extern_producer.cpp

```diff
@@ -13,13 +12,17 @@ using namespace Halide;
 // Some helper functions for rounding
 int round_down(int, int);
 int round_up(int x, int m) {
-    if (x < 0) return -round_down(-x, m);
-    else return ((x + m - 1) / m) * m;
+    if (x < 0)
```

Comment:
braces

Reply:
(sadly, clang-format won't attempt to enforce this AFAICT -- we'd have to use clang-tidy for that)